### DISCUSSION TITLE
A few questions

### DISCUSSION THREAD

**Discussion by patrickli** [revk/xESP32-Faikout#33] [Q&A] [answered]
Hi guys,

I have a multi-unit system with 4x indoor units. 3x FTXS20DVMA and 1x FTXS50FVMA. I would like to use this to make them controllable via Home Assistant. I have 2 questions.

* These are pretty old systems (made in 2008 I believe) but they do have S21 ports on the main PCB. Do you think they are compatible?
*  Where can I buy those wire harnesses to connect the adapter to the PCB? I do have a bunch of breadboard jumpers and they look like they would fit.

You advice is very much appreciated. Thanks.

**Comment by revk** [comment]
My understanding is S21 is enough of a standard for Daikin that it should. But not really something I could guarantee. Maybe get one and try.

I don't know where you get the wiring, sorry.

**Reply by patrickli** [reply]
Alright. Ordered one. Let's see if it works when I get it in a few weeks.

**Comment by patrickli** [comment]
Well you know what, it works like a charm. I installed this in the FTXS20DVMA which is manufactured in 2008 (15 years old!!). Just used some jumpers and paper tape. No official harness/cable required. 

But you are out of stock. Do you know when more will be available?

Some photos

![PXL_20230530_023547251](https://github.com/revk/ESP32-Faikin/assets/502326/05556090-d572-4342-9776-39cf1c4f564a)
![PXL_20230530_023705203](https://github.com/revk/ESP32-Faikin/assets/502326/d8d8b047-bd51-477c-8b7d-4baf2d525aa0)
![PXL_20230530_030901554_exported_1797_1685416169084](https://github.com/revk/ESP32-Faikin/assets/502326/dd442268-7f5d-4040-a79d-d53b1c4d5f7a)

**Reply by revk** [reply, marked as answer]
New listing https://www.amazon.co.uk/dp/B0C2ZYXNYQ which has the 5 pin connector (JST EH type). I expect your header wires will work though.

**Reply by patrickli** [reply]
Oh btw, this PCB I bought, Amazon is able to ship it directly to New Zealand. I think it is the old version. Did you do anything to make it able to ship internationally? Might worth checking as it is way more convenient.

**Reply by revk** [reply]
I did not do anything special to allow it to be shipped internationally, in fact I tried to get an explanation from Amazon as to why they would not, and got nowhere. I have failed to get any explanations from Amazon, sorry. Best option may be a parcel forwarding company.

**Reply by JeanOlivier** [reply]
I was able to order your old purple PCB twice to Canada very recently simply through Amazon. But you new one does appear as not possible to export. Weird.

**Reply by revk** [reply]
Anyone that can work out the whims on Amazon is better than I. I'll look again though.

**Comment by PaskoMoto** [comment]
Hi.

I will try to connect my first Faikin card again tomorrow. 
I'm planning to use jumpers as shown in this discussion.
The PCB shown in this discussion looks very much like the one I'm planning to connect the Faikin to. 
However, looking at the pictures in this discussion, I have a doubt:
Where does the yellow and subsequent cables connect to the **new Faikin boards which have 5 instead of 4 pins**?
Will it still connect to the first pin starting to count from the right or will it go in the second one?

In other words:
Which of the Faikin pins must not be connected? 
The right most one or the left most one?

Thanks in advance.
Best Regards.
